src/menu/menu.less

```diff
@@ -70,3 +73,17 @@
         margin: 4px 0;
     }
 }
+
+.reactist_menulist [role='menuitem'] .reactist_menuitem_icon {
```

Comment:
Because I plan to eventually move this file to CSS modules without the `.less` preprocessing. Also, when that happens, I'll remove the nesting, but also the prefix elements in this selector, because of how CSS classes in CSS modules are not global, nesting is not needed as much.